app/dts/layouts/common/tkl.dtsi

```diff
@@ -0,0 +1,101 @@
+#include <physical_layouts.dtsi>
```

Comment:
Maybe not worth including until something needs them, but there are several TKL variants. Specifically, every combination of
- ANSI or ISO
- winkey (most common) or winkeyless
- 87 (most common) or 88 key (adds F13)

We may at least want to decide on how all of those variants should be named up front.

Reply:
For all the other layouts i've named them as `layout_<size>_<quirks/specifics>`, I think that can be extended to this too e.g. `layout_tkl_f13ansi`